src/platform/imx8/lib/memory.c

```diff
@@ -86,7 +86,7 @@ static SHARED_DATA struct mm memmap = {
 		.size = HEAP_BUFFER_SIZE,
 		.info = {.free = HEAP_BUFFER_SIZE,},
 		.caps = SOF_MEM_CAPS_RAM | SOF_MEM_CAPS_CACHE |
-			SOF_MEM_CAPS_DMA,
+			SOF_MEM_CAPS_DMA | SOF_MEM_CAPS_HP | SOF_MEM_CAPS_LP,
```

Comment:
I don't like this change on the imx8 platforms, since these heaps are actually not LP-capable. We may add some actual LP ranges in the future. If DRAM turns off then these heaps are dead.

Reply:
@paulstelian97 ack. thats why i marked it as "Do not merge". It was merely intended to start a discussion on how to address the issue we're seeing on our APL nocodec topology with dynamic pipelines.